core/renderers/common/constants.js

```diff
@@ -571,6 +569,60 @@ Blockly.blockRendering.ConstantProvider.prototype.refreshTheme = function(
   }
 };
 
+/**
+ * Set dynamic constants that depent on other values or theme properties.
```

Comment:
I wouldn't worry about updating this PR but I think the [latest style recommendation](https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments) is third person verbs for method descriptions.

Reply:
Thanks